### PR TITLE
Feature/board acceptance

### PR DESCRIPTION
**Pull request by raulchoque-eng**
create the CRUD's for feature Board

### CODE REVIEW

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 1]
```diff
@@ -61,8 +62,9 @@ public void iSetUpData(String data) {
     @When("I send the request")
     public void sentRequest() {
         response = requestManager.makeRequest();
-        String idBoard = response.body().jsonPath().get("id");
-        board.setId(idBoard);
+        String id = response.body().jsonPath().get("id");
+        context.setId(id);
+
```
Comment (MadayAlcala) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java]: Please delete unnecessary line.

Comment (raulchoque-eng) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java]: Thanks, this mistake was fixed.

**Review by MadayAlcala — APPROVED** [review]

File: src/test/java/com/foundation/trello/hook/BoardHook.java [hunk 2]
```diff
@@ -15,29 +15,46 @@
...
+        String endPoint = "/boards/".concat(context.getId());
         String method = "delete";
         requestManager = FactoryRequest.getRequest(method);
         requestManager.setMethod(method);
         requestManager.setEndPoint(endPoint);
         Response response = requestManager.makeRequest();
         Log.getInstance().getLog().info(response);
+
+    }
+    /**
```
Comment (LimbertVargas) [inline src/test/java/com/foundation/trello/hook/BoardHook.java:45]: please, add a space between the comment and the brace.

File: src/test/resources/feature/board/board_acceptance.feature [hunk 3]
```diff
@@ -12,3 +12,28 @@ Feature: Acceptance test of "board", CRUD's(Create, Read, Update and Delete)
...
+      """
+    When I send the request
+    Then I get a 200 status code as response
+
+  @create-board
+  Scenario: Delete a board by Id
+    Given I create a "DELETE" request to "/boards/{idBoard}" endpoint
+    When I send the request
+    Then I get a 200 status code as response
+
```
Comment (LimbertVargas) [inline src/test/resources/feature/board/board_acceptance.feature:39]: please fix this message adding a final space

Comment (raulchoque-eng) [reply src/test/resources/feature/board/board_acceptance.feature:39]: Okay, that mistake was fixed

File: src/test/java/com/foundation/trello/hook/BoardHook.java (hunk 2 above)
Comment (raulchoque-eng) [reply src/test/java/com/foundation/trello/hook/BoardHook.java:45]: Okay, that mistake was fixed

**Review by MadayAlcala — APPROVED** [review]

**Review by LimbertVargas — APPROVED** [review]

**Pull request merged**